teps/0128-scheduled-runs.md

```diff
@@ -63,7 +64,100 @@ If the EventListener is exposed outside of the cluster, the user must also sign
 - Release Pipeline: User would like to create releases of their project on a nightly basis.
 
 ## Proposal
+Create ScheduledRun CRD with `resourcetemplates` and `schedule` field
 
+This solution doesn't use TriggerTemplate or TriggerBindings. Instead, it's spec has `resourcetemplates` which can have one or more pipelineruns, taskruns or other allowed Kubernetes and Tekton resources.
+
+```yaml
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: ScheduledRun
+metadata:
+  name: nightly-release
+spec:
+  cloudEventSink: http://eventlistener.free.beeceptor.com
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      generateName: simple-pipeline-run-
+    spec:
+      pipelineRef:
+        name: simple-pipeline
+      params:
+      - name: message
+        value: "Hi"
+      - name: git-url
+        value: "https://github.com/tektoncd/community.git"
+      workspaces:
+      - name: git-source
+        emptyDir: {}
+    schedule: "0 0 * * *"
+```
+
+`ScheduledRun` reconciler which creates resources can be part of Triggers but we can move it to Pipelines also.
```

Comment:
What's the reason to move this to Pipelines? It seems to me like it would make the most sense to keep in Triggers.

Reply:
Yes, after the discussion it makes sense to keep in Triggers. Updating the doc with that.